@here/harp-omv-datasource/lib/OmvDataSource.ts

```diff
@@ -252,6 +252,7 @@ export class OmvDataSource extends TileDataSource<OmvTile> {
             return;
         }
         this.configureDecoder(styleSet, languages, this.m_decoderOptions);
+        this.mapView.markTilesDirty(this);
```

Comment:
Is there a reason why we overwrite `setStyleSet`? Looking at `OmvDataSource.configureDecoder` it looks exactely the same as `TileDataSource.setStyleSet` except for the `options` but since OptionsMap is not related to `OmvDataSource` we should just move it to `TileDataSource`

Reply:
Agree, will update fix.